packages/http-server-csharp/src/interfaces.ts

```diff
@@ -101,7 +101,7 @@ export class NumericValue extends CSharpValue {
     this.value = value;
   }
   public emitValue(scope?: Scope<string> | undefined): string {
-    return `${this.value}` ?? "0";
+    return `${this.value}`;
```

Comment:
This matches current state - wonder if this was meant to be `${this.value ?? 0}`